tests/unit/date/helper.js

```diff
@@ -0,0 +1,30 @@
+define( [
+	"jquery",
+	"globalize",
+	"globalize/date",
+	"lib/helper"
```

Comment:
If the intent is to add a `getAttributes` method to the helper for these tests, then this is incorrect. @rxaviers' mapping in his most recent comment shows what's actually happening, so you're modifying `globalize/date` instead of `lib/helper`.